### PR TITLE
doc: make readme and values file simpler

### PR DESCRIPTION
**Pull request by deontpearson**
## Description

The readme and `values.example.yaml` files were drifting. This PR removes the content of the readme that can change often. Using the `values.example.yaml` file also helps to illustrate how the values file should look.

## Types of changes

_Put an `x` in the boxes that apply._

- [ ] Bugfix (`fix`): A non-breaking change which fixes an issue or bug
- [ ] New Feature (`feat`): A non-breaking change which adds a new feature or new functionality
- [ ] Tests (`test`): Adding missing tests or correcting existing tests
- [ ] Refactoring (`refactor`): A code change that neither fixes a bug nor adds a feature, e.g Deleting unused or redundant code
- [x] Documentation (`docs`): Documentation only changes
- [ ] Other (`style`, `build`, `ci`, `performance`): Formatting, linting or styling. Changes to the CI pipeline. Performance improvements. Changes that affect the build system.

https://uxi.atlassian.net/browse/BEP-737

### CODE REVIEW

File: charts/aruba-uxi/values.example.yaml [hunk 1]
```diff
@@ -1,177 +1,167 @@
...
+        ## Configures the service type that is created (optional, default: ClusterIP)
+        type: ClusterIP
+        ## Configures the service port to expose (optional, default: 80)
+        port: 80
+      ## Enables the liveness probe to determine if the pod started up correctly (optional, default: {})
       livenessProbe:
+        ## Toggles the liveness probe (optional, default: true)
         enabled: true
+        ## The API path to query for liveness testing (optional, default: /livez)
         path: /livez
```
Comment (MaraisKruger) [inline charts/aruba-uxi/values.example.yaml:92]: If we plan to make /health/* the default from now on, should we not document this here?

Comment (MaraisKruger) [reply charts/aruba-uxi/values.example.yaml:92]: And change the default as well

File: charts/aruba-uxi/values.example.yaml [hunk 2]
```diff
@@ -181,402 +171,158 @@ aruba-uxi:
...
-  ## The key is used as the name of the cronjob and subsequent objects created
-  ##
+      ## Configures the pod update strategy. The Deployment updates Pods in a rolling update fashion when .spec.strategy.type==RollingUpdate (optional, default: {})
+      updateStrategy:
+        # specifies the maximum number of Pods that can be unavailable during the update process.
+        # The value can be an absolute number (for example, 5) or a percentage of desired Pods (for example, 10%).
+        maxSurge: 25%
+        # specifies the maximum number of Pods that can be created over the desired number of Pods.
+        # The value can be an absolute number (for example, 5) or a percentage of desired Pods (for example, 10%).
+        maxUnavailable: 25%
```
Comment (MaraisKruger) [inline charts/aruba-uxi/values.example.yaml:258]: These comments have been swapped around

Comment (deontpearson) [reply charts/aruba-uxi/values.example.yaml]: nice catch

**Review by MaraisKruger — APPROVED** [review]

**Pull request merged**